.github/scripts/generate_ci_workflows.py

```diff
@@ -667,6 +667,15 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
             labels=set([LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_LINUX, LABEL_CIFLOW_CUDA]),
         ),
     ),
+    CIWorkflow(
```

Comment:
I don't believe we actually need to do this anymore

Reply:
We do, otherwise it fails shellcheck lint

Reply:
Not needed after rebase